tests/test_api.py

```diff
@@ -28,7 +28,7 @@ def test_property_result_is_str(func: str) -> None:
 
 def test_method_result_is_path(func_path: str) -> None:
     method = getattr(platformdirs, func_path)
-    result = method("MyApp", "MyCompany")
+    result = method()
```

Comment:
seems unrelated change

Reply:
I was asked to remove the arguments from get_user_documents_dir, since it doesn't use them. The tests would fail here without the change.